sympy/printing/tests/test_python.py

```diff
@@ -43,6 +43,9 @@ def test_python_basic():
         "y = Symbol('y')\nx = Symbol('x')\ne = 1 - 3*y/(2*x)"]
 
     # Multiplication
+    from sympy import Mul, Pow
+    assert python(Mul(-2, x, Pow(Mul(y,y,evaluate=False), -1, evaluate=False)
```

Comment:
Why did you add a test to the Python printer? The fix is in the str printer, so the test should go in the str printer tests.